hasher-matcher-actioner/hmalib/common/models/bank.py

```diff
@@ -678,10 +678,12 @@ def update_bank_member(
 
     def remove_bank_member(self, bank_member_id: str):
         """
-        Removes the bank member and associated signals from the bank. Merely
-        marks as removed, does not physically delete from the store. DOES NOT
-        stop matching until index is updated. DOES NOT undo any actions already
-        taken.
+        Removes the bank member from the bank. Merely marks as removed, does not
```

Comment:
Nasty bug. Documentation used to say that associated signals are removed, when they are not.

Reply:
What's the expected long term behavior here? For a multitude of reasons, I assume we want these to be scrubbed from the bank at some point.

Reply:
The recommended way is to use bank_operations.remove_bank_member which does the coordination for you. I wanted to keep table operations atomic as much as possible in this class.